base/iterators.jl

```diff
@@ -934,6 +934,28 @@ IteratorSize(::Type{<:DropWhile}) = SizeUnknown()
 eltype(::Type{DropWhile{I,P}}) where {I,P} = eltype(I)
 IteratorEltype(::Type{DropWhile{I,P}}) where {I,P} = IteratorEltype(I)
 
+"""
+    findall(f, it)
```

Comment:
```suggestion
    findall(f, it)
    findall(it)
```
To be consistent with the Base version